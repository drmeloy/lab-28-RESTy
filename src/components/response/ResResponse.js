import React from 'react';
import PropTypes from 'prop-types';
import ReactJson from 'react-json-view';
import styles from './Response.css';

const ResResponse = ({ responseBody }) => {
  console.log(responseBody);

  return (
    <details >
      <summary>"Response" : </summary>
      <ReactJson src={responseBody} theme="solarized" name={false} collapsed="2" displayDataTypes={false} iconStyle="circle" enableClipboard={false} />
    </details>
  );
}

ResResponse.propTypes = {

};

export default ResResponse;
