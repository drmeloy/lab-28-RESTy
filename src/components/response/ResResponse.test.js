import React from 'react';
import { shallow } from 'enzyme';
import ResResponse from './ResResponse';

describe('ResResponse component', () => {
  it('renders ResResponse ', () => {
    const wrapper = shallow(<ResResponse responseBody={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
