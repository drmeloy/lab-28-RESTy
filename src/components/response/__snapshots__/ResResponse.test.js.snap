// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResResponse component renders ResResponse  1`] = `
<details
  open={false}
>
  <summary>
    "Response" :
  </summary>
  <t
    collapseStringsAfterLength={false}
    collapsed="2"
    defaultValue={null}
    displayDataTypes={false}
    displayObjectSize={true}
    enableClipboard={false}
    groupArraysAfterLength={100}
    iconStyle="circle"
    indentWidth={4}
    name={false}
    onAdd={false}
    onDelete={false}
    onEdit={false}
    onSelect={false}
    shouldCollapse={false}
    sortKeys={false}
    src={Object {}}
    style={Object {}}
    theme="solarized"
    validationMessage="Validation Error"
  />
</details>
`;
